api/v1alpha1/verifier_types.go

```diff
@@ -34,6 +34,9 @@ type VerifierSpec struct {
 	// # Optional. URL/file path
 	Address string `json:"address,omitempty"`
 
+	// OCI Artifact source to download the plugin from, optional
+	Source string `json:"source,omitempty"`
```

Comment:
I added `source` here as a new field in the PoC

There might be an opportunity to improve this and reduce confusion versus `address`, which is actually the **directory** where the binary is expected to be located

Reply:
agreed, maybe we can also clarify the intention for each property in docs